release/pipline/dags/istio_common_dag.py

```diff
@@ -174,7 +177,15 @@ def GenerateTestArgs(**kwargs):
     git config --global user.email "testrunner@istio.io"
     git clone {{ settings.MFEST_URL }} green-builds || exit 2
     cd green-builds
-    git checkout {{ settings.MFEST_COMMIT }} || exit 3
+    SHA=`grep {{ settings.MFEST_FILE }} | cut -f 6 -d \\"` || exit 3
+    cd ..
+    git clone {{ settings.ISTIO_REPO }} istio-code
+    cd istio-code/release
+    git checkout ${SHA} || exit 4
+    gsutil cp *.sh gs://{{ settings.GCS_RELEASE_TOOLS_PATH }}/data/release/
```

Comment:
I wouldn't be opposed to copying everything.  Currently the only thing that's not copied is the OWNERS and README.md.  "cp" would skip the "pipline" directory (shouldn't this be "pipeline"?) and I'm assuming/hoping that "gsutil cp" would do the same.

Reply:
people can any other types of file at any time. GCS is weird and the gsutill cp is different then posix cp so I don't want to take any changes.